saleor/plugins/webhook/tasks.py

```diff
@@ -367,6 +374,7 @@ def send_webhook_request_sync(app_name, delivery):
         delivery_update(delivery, EventDeliveryStatus.FAILED)
         raise ValueError("Unknown webhook scheme: %r" % (parts.scheme,))
     delivery_update(delivery, response.status)
+    report_event_delivery_attempt(delivery.event_type, attempt)
```

Comment:
Shouldn't we provide the response status code and contents?

Reply:
Response headers and content is stored in the attempt, which is passed here. This response data is then used to report this webhook delivery attempt. That is an excellent point because we skipped storing response status codes in a delivery attempt. Keeping a status code can be helpful in the future, and maybe we should add it to the `EventDeliveryAttempt` data model @maarcingebala @jakubkuc

Reply:
Yep, I think it would be helpful to store it. We could use a new field in the attempt model for that.